modules/understanding-vulnerability-scores.adoc

```diff
@@ -6,26 +6,20 @@
 = Understanding vulnerability scores
 
 [role="_abstract"]
-In the RHACS portal shows a single Common Vulnerability Scoring System (CVSS) base score for each vulnerability.
+The {product-title-short} portal shows a single Common Vulnerability Scoring System (CVSS) base score for each vulnerability.
 {product-title} shows the CVSS score based on the following criteria:
 
 * If a CVSS v3 score is available, {product-title} shows the score and lists `v3` along with it.
 For example, `6.5 (v3)`.
 +
 [NOTE]
 ====
-CVSS v3 scores are only available if you are using Scanner version 1.3.5 and newer.
+CVSS v3 scores are only available if you are using the StackRox Scanner version 1.3.5 and newer or Scanner V4.
 ====
 * If a CVSS v3 score is not available, {product-title} shows only the CVSS v2 score.
 For example, `6.5`.
 
 You can use the API to get the CVSS scores.
-If CVSS v3 information is available for a Common Vulnerabilities and Exposures (CVE), the response includes both CVSS v3 and CVSS v2 information.
+If CVSS v3 information is available for a vulnerability, the response includes both CVSS v3 and CVSS v2 information.
```

Comment:
Sorry just noticed this: CVSS v2 is pretty much not produced anymore, so it's very possible we only get v3 now. Maybe replace "the response includes both" with "the response may include both"